docs/content/developers/writing-style-guide.md

```diff
@@ -0,0 +1,222 @@
+
+Similarly, “just do X” suggests that “X” should be easy or obvious. Most of the time “just” can be omitted and everyone wins.
+
+If you’d like to reassure the reader something is easy, illustrate it with a demonstration and let them draw their own conclusion!
+
+| Write This 👍 | Not This ❌
+| -- | --
+| Change your project’s PHP version by either editing `.ddev/config.yaml` to set `php_version: "8.2"`, or by running `ddev config --php-version=8.2`, followed by running `ddev restart`. | It’s easy to change your project’s PHP version! Just edit your project’s `.ddev/config.yaml` to set `php_version: "8.2"`, or run `ddev config --php-version=8.2`, followed by running `ddev restart`. |
+
+## Writing Style
+
+DDEV’s documentation should be consistent throughout, which benefits both the reader taking in information and the contributor looking for examples to follow.
+
+!!!tip "Read It Aloud"
+    If you get tripped up speaking your words out loud, someone else will get tripped up reading them, too.
+
+### Use Correct Capitalization and Punctuation
+
+Write with appropriate grammar and style for U.S. English, including capitalization and punctuation. Variations in spelling and writing style make the documentation harder to read, and we want to be respectful of the reader’s time and attention.
+
+| Write This 👍 | Not This ❌
+| -- | --
+| Let’s not capitalize random words for emphasis. | Let’s not Capitalize random Words for emphasis.
+| That organization uses a lovely American color. | That organisation uses a lovely American colour.
+| We can use “curly quotes” now that we’re post-typewriter. | We can use "curly quotes" now that we're post-typewriter.
+
+### “Run” Commands
+
+We “run” commands. We don’t “do” them, and the command itself is not a verb. Whenever possible, reinforce that a given thing in backticks is intended as a console command by using the word “run” before it.
+
+| Write This 👍 | Not This ❌ |
+| -- | -- |
+| Run `ddev config` to set up your project. | Do `ddev config` to set up your project.<br>You can `ddev config` to set up your project. |
+| If you get stuck, run `ddev restart`. | If you get stuck, just `ddev restart`. |
+
+### Use Active Third Person
+
+Avoid impersonal language featuring unknown individuals or shadowy organizations.  
+“It is recommended,” for example, could be a warmer “we recommend” or “Laravel users recommend”.
+
+Write on behalf of the community and not yourself—use “we” and not “I”.
+
+| Write This 👍 | Not This ❌ |
+| -- | -- |
+| We recommend Colima for the best performance. | It is recommended for performance that you use Colima. |
+| Some prefer Redis for runtime caches. | I like using Redis for runtime caches. |
+
+### Write Once and Link
+
+Try to keep from repeating yourself in the documentation. Instead, write carefully and link to that well-crafted specimen, whether it’s across the page or off to another section. This has two benefits:
```

Comment:
Absolutely correct, but there are times that links with necessary info distract from the flow, so it's better to just tell them the missing info.

Reply:
The realist in me probably agrees, but I have lists of redundancies to go through and address—both linking to canonical explanation and adding the _ability_ to link it in the first place. (Like individual commands that live in a table, for example.)